### PR TITLE
ISPN-6978 AffinityIndexManager does not work on transactional caches

### PR DESCRIPTION
**Pull request by gustavonalle**
https://issues.jboss.org/browse/ISPN-6978

### CODE REVIEW

File: query/src/test/java/org/infinispan/query/blackbox/ClusteredCacheWithAffinityIndexManagerTxTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.infinispan.query.blackbox;
+
+import org.testng.annotations.Test;
+
+/**
+ * @since 9.0
+ */
+@Test(groups = "functional", testName = "query.blackbox.ClusteredCacheWithAffinityIndexManagerTxTest")
+public class ClusteredCacheWithAffinityIndexManagerTxTest extends ClusteredCacheWithAffinityIndexManagerTest {
```
Comment (anistor) [inline query/src/test/java/org/infinispan/query/blackbox/ClusteredCacheWithAffinityIndexManagerTxTest.java]: This is added in first commit then renamed in the second. I'd squash those two.

Comment (gustavonalle) [inline query/src/test/java/org/infinispan/query/blackbox/ClusteredCacheWithAffinityIndexManagerTxTest.java]: np

**Comment by anistor** [comment]
Looks good. I'll squash and pull it soon, just waiting for another run.

**Comment by anistor** [comment]
Integrated in master. Thanks @gustavonalle !

**Pull request closed without merging**